src/Behat/Behat/Output/Node/EventListener/JUnit/JUnitFeatureElementListener.php

```diff
@@ -154,21 +117,15 @@ private function captureStepEvent(Event $event)
     }
 
     /**
-     * Prints the feature on AFTER event.
+     * Captures scenario tested event.
      *
      * @param Formatter $formatter
-     * @param Event     $event
+     * @param ScenarioTested $event
```

Comment:
adding a phpdoc that does not match the parameter type does not make sense

Reply:
fixed